### PR TITLE
Optimizer disaster recovery docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added, or deleted? (Required)
Optimizer disaster recovery docs

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/recover-deleted-cluster.md [hunk 1]
```diff
@@ -6,25 +6,25 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/recover-deleted-cluster/']
...
+此时你可以使用 `kubectl create` 命令来创建一个同名同配置的集群，之前保留下来未被删除的 PV/PVC 以及数据会被复用：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 kubectl -n ${namespace} create -f tidb-cluster.yaml
 ```
 
-## Helm 管理的集群意外删除后恢复
+## 恢复在 Helm 上误删的 TiDB 集群
```
Comment (DanielZhangQD) [inline zh/recover-deleted-cluster.md:23]: 这部分 Helm 相关的删掉吧，从 1.1 开始这种方式就 deprecate 了

Comment (TomShawn) [reply zh/recover-deleted-cluster.md]: Done

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Pull request merged**